src/common-types.ts

```diff
@@ -79,9 +82,8 @@ export interface PythonKeys {
 }
 
 export interface UsableData {
-    category: string;
-    node: string;
     id: string;
+    identifier: string;
```

Comment:
We should probably call `identifier` `typeId` or `schemaId`. This will be VERY confusing otherwise.

Reply:
SchemaId is good. I couldn't think of a good name to call it. I'll fix that